### ISSUE TITLE
Accessibility improvements, various

### ISSUE THREAD

**Issue by anevins12** [h5p/h5p-questionnaire#24]
I can see that there's been great effort into handling the keyboard focus of the questionnaire, but I can also see areas that could be improved:

- Using a `<fieldset>` to group the question and answer and then using a `<legend>` to wrap the question
- Using an underline to on the active step number to indicate its state as well as its current orange colour && Adding hidden text to explain that "1" is the active step
- Using `aria-label` for fields that are missing an associated label element, such as the `<textarea>` field
